kubernetes/e2e_test/test_utils.py

```diff
@@ -17,100 +17,282 @@
+        self.assertEqual("extensions/v1beta1",
+                         k8s_api.get_api_resources().group_version)
+        dep = k8s_api.read_namespaced_deployment(name="nginx-deployment",
+                                                 namespace="default")
         self.assertIsNotNone(dep)
-        resp = k8s_api.delete_namespaced_deployment(
-            name="nginx-deployment", namespace="default", 
+        k8s_api.delete_namespaced_deployment(
+            name="nginx-deployment", namespace="default",
             body={})
-    
+
     def test_core_pod_yaml(self):
+        """
+        Test for creating and deleting a pod on default namespace.
+        Using core API
+        Using yaml file core-pod.yaml
+        """
         k8s_client = client.api_client.ApiClient(configuration=self.config)
-        k8s_api = utils.create_from_yaml(k8s_client, 
-            "kubernetes/e2e_test/test_yaml/core-pod.yaml")
-        self.assertEqual("v1", 
-            k8s_api.get_api_resources().group_version)
-        pod = k8s_api.read_namespaced_pod(name="myapp-pod", 
-            namespace="default")
+        k8s_api = utils.create_from_yaml(
+            k8s_client, "kubernetes/e2e_test/test_yaml/core-pod.yaml")
+        self.assertEqual("v1",
+                         k8s_api.get_api_resources().group_version)
+        pod = k8s_api.read_namespaced_pod(name="myapp-pod",
+                                          namespace="default")
         self.assertIsNotNone(pod)
-        resp = k8s_api.delete_namespaced_pod(
+        k8s_api.delete_namespaced_pod(
             name="myapp-pod", namespace="default",
             body={})
 
     def test_core_service_yaml(self):
+        """
+        Test for creating and deleting a service on default namespace.
+        Using core API
+        Using yaml file core-service.yaml
+        """
         k8s_client = client.api_client.ApiClient(configuration=self.config)
-        k8s_api = utils.create_from_yaml(k8s_client, 
-            "kubernetes/e2e_test/test_yaml/core-service.yaml")
-        self.assertEqual("v1", 
-            k8s_api.get_api_resources().group_version)
+        k8s_api = utils.create_from_yaml(
+            k8s_client, "kubernetes/e2e_test/test_yaml/core-service.yaml")
+        self.assertEqual("v1",
+                         k8s_api.get_api_resources().group_version)
         svc = k8s_api.read_namespaced_service(name="my-service",
-            namespace="default")
+                                              namespace="default")
         self.assertIsNotNone(svc)
-        resp = k8s_api.delete_namespaced_service(
+        k8s_api.delete_namespaced_service(
             name="my-service", namespace="default",
             body={})
-        
+
     def test_core_namespace_yaml(self):
+        """
+        Test for creating and deleting namespace
+        Using core API
+        Using yaml file core-namespace.yaml
+        """
```

Comment:
def test_create_namespace_from_yaml(self):
        """
        Should be able to create a namespace.
        """